modelina-cli/src/helpers/java.ts

```diff
@@ -18,6 +18,13 @@ export const JavaOclifFlags = {
     required: false,
     default: false
   }),
+  javaArrayType: Flags.string({
+    type: 'option',
+    description: 'Java specific, define which type of array needs to be generated.',
+    options: ['Array', 'List'],
```

Comment:
```suggestion
    options: ['array', 'list'],
```

Lets keep it to all lower case or all upper case options for now :v: